src/server/plugins/engine/components/YesNoField.ts

```diff
@@ -37,34 +33,15 @@ export class YesNoField extends ListFormComponent {
     ]
   }
 
-  get items() {
-    return this.list?.items ?? []
-  }
-
-  get values() {
-    return [true, false]
-  }
-
   constructor(def: YesNoFieldComponent, model: FormModel) {
     super(def, model)
```

Comment:
@davidjamesstone I've let the Yes/No component inherit both schema from `ListFormComponent`

Do you know why it's casting `true` → `'true'` despite the boolean list type?

Reply:
The Yes/No list is already in the FormModel so this works:

```patch
  constructor(def: YesNoFieldComponent, model: FormModel) {
-   super(def, model)
+   super({ ...def, list: '__yesNo' }, model)
```